Formula/mrboom.rb

```diff
@@ -0,0 +1,23 @@
+class Mrboom < Formula
+  desc "Mr.Boom is an 8 player Bomberman clone"
+  homepage "http://mrboom.mumblecore.org"
+  url "https://github.com/Javanaise/mrboom-libretro/archive/3.8.tar.gz"
+  sha256 "2c05a68f0d513013e28934c3ef8f3b95f5330790ca2c8bd05a5413c3d5edd087"
+
+  depends_on "cmake" => :build
+  depends_on "libmodplug"
+  depends_on "lzlib"
+  depends_on "minizip"
+  depends_on "sdl2"
+  depends_on "sdl2_mixer"
+
+  def install
+    system "make", "mrboom", "LIBSDL2=1"
+    bin.install "mrboom.out" => "mrboom"
+    man6.install "Assets/mrboom.6"
+  end
+
+  test do
+    system "#{bin}/mrboom", "--version"
```

Comment:
we'll need a test that exercises and verifies the functionality of the software

Reply:
It does, it's checking decrunching the data and modplug when running version.

Reply:
something that loads a config file or similar would work. `--version` and `--help` tests aren't allowed for new formulae

Reply:
OK then I should simply remove it

Reply:
You must have a test block. Just not one that does `--version` or `--help`.